tests/cli/test_compose_sanity.sh

```diff
@@ -16,7 +16,38 @@ rlJournalStart
     rlPhaseEnd
 
     rlPhaseStartTest "compose start"
-        UUID=`$CLI --test=2 compose start example-http-server tar`
+        UUID=`$CLI compose start example-http-server ami`
+        rlAssertEquals "exit code should be zero" $? 0
+        UUID=`echo $UUID | cut -f 2 -d' '`
+
+        if [ -n "$UUID" ]; then
+            until $CLI compose details $UUID | grep 'RUNNING'; do
+                sleep 20
+                rlLogInfo "Waiting for compose to start running..."
+                if $CLI compose info $UUID | grep 'FAILED'; then
+                    rlFail "Compose FAILED!"
+                    break
+                fi
+            done;
+        else
+            rlFail "Compose UUID is empty!"
+        fi
+
+        # check if anaconda is really running
+        until ps -axo comm,pid | grep '^anaconda'; do
+            sleep 10
+            rlLogInfo "Waiting for anaconda to start running..."
+        done;
+    rlPhaseEnd
+
+    rlPhaseStartTest "cancel compose"
+        rlRun -t -c "$CLI compose cancel $UUID"
+        rlRun -t -c "$CLI compose details $UUID" 1 "compose is canceled"
+        rlAssertNotExists "/var/run/anaconda.pid"
```

Comment:
IMO remove this line and don't change anything else.

Reply:
I agree, this line is not really necessary for covering the intended test scenario.